### ISSUE TITLE
`sortText` not set on property completions if triggered without typing

### ISSUE THREAD

**Issue by mjbvz** [microsoft/TypeScript#30013]
**TypeScript Version:**  3.4.0-dev.20190220

**Search Terms:**
- completions
- suggestions
- sortText

**Code**
For the TypeScript code:

```ts
class Foo {
    prop: number;

    render() {
        
    }
}
```

Trigger suggestions in `render` using ctrl+space instead of typing

**Expected behavior:**
The suggestions for `prop` and `render` should have a higher `sortText` value. This is what we see when suggestions are triggered after you start typing:

```ts
class Foo {
    prop: number;

    render() {
        t|
    }
}
```

**Actual behavior:**
The `sortText` is `"0"` for all suggestions, including those for properties

**Playground Link:** 

**Related Issues:**

**Comment by sheetalkamat** [comment]
@mjbvz I don't see any difference between getting completions with `ctr+space` or typing. 
The sort text issue is #15024 
[tsserver.log](https://github.com/microsoft/TypeScript/files/3188991/tsserver.log)

**Comment by sheetalkamat** [comment]
So duplicate of #15024 ?

**Comment by mjbvz** [comment]
Yes, this falls under #15024
